src/graphene_set.h

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2018 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_GRAPHENE_SET_H
+#define BITCOIN_GRAPHENE_SET_H
+
+#include "bloom.h"
+#include "boost/dynamic_bitset.hpp"
+#include "hash.h"
+#include "iblt.h"
+#include "random.h"
+#include "serialize.h"
+#include "uint256.h"
+#include "util.h"
+
+#include <atomic>
+#include <cmath>
+#include <vector>
+
+using namespace std;
```

Comment:
we decided to avoid implicit use of `std` namespace, so that we are going to be explicit in any declaration involving `std` type. So always use `std::vector<size_t> v;` rather than `using namespace std;` and later on `vector<size_t> v`.

I noticed that you use explicit declaration later on (e.g. https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/973/commits/9af45973c770e0edb70c2a13500148be0b2a5b71#diff-c18e3d2f8ec69646ca22a86494be587aR29)

Reply:
@sickpig will fix. I was wondering about the convention. I've been pretty sloppy about it up to this point.